package.json

```diff
@@ -86,26 +86,29 @@
     "stylelint-config-nebenan": "^2.0.0",
     "ts-loader": "^9.4.1",
     "tslib": "^2.4.1",
-    "typescript": "^4.8.4",
-    "updeep": "^1.2.0",
+    "typescript": "^4.9.3",
+    "updeep": "^1.2.1",
     "val-loader": "^2.1.2",
     "webpack": "^5.36.2",
     "webpack-cli": "^4.7.2",
     "webpack-dev-middleware": "^4.3.0",
-    "webpack-hot-middleware": "^2.25.0",
+    "webpack-hot-middleware": "^2.25.3",
     "webpack-merge": "^5.7.3"
   },
   "scripts": {
-    "postinstall": "lerna bootstrap && link-parent-bin && link-parent-bin -c package-templates && yarn run tryBuild",
-    "check": "npm run lint && npm run test",
     "reinstall": "rm -rf node_modules yarn.lock && yarn",
     "ci": "yarn install --frozen-lockfile",
     "build": "lerna exec --parallel --no-bail yarn run build",
     "tryBuild:preview": "lerna exec --parallel --no-bail yarn run build:preview || true",
     "tryBuild": "lerna exec --parallel --no-bail yarn run build || true",
-    "lint": "npm run eslint && npm run stylelint",
-    "eslint": "eslint -f stylish \"./**/*.{js,jsx,ts,tsx}\" ",
-    "stylelint": "stylelint \"./**/*.scss\" ",
-    "test": "scripts/run-tests"
+    "--- CHECKS AND TEST SCRIPTS ---": "-----------------------------",
+    "check": "npm test && npm run lint && echo \"\\033[0;32mAll checks pass\\033[0m\"",
+    "lint": "npm run lint:javascript && npm run lint:styles && npm run lint:typescript",
+    "lint:javascript": "eslint -f stylish \"./**/*.{js,jsx,ts,tsx}\"",
+    "lint:typescript": "scripts/check-typescript",
```

Comment:
Still has the same issue as before https://github.com/goodhood-eu/goodhood/pull/178#discussion_r1028972290

I think we need this:
```
lerna exec --parallel --no-bail yarn run lint:typescript
```

This would:
- Go into each package
- Execute lint:typescript there 
- Wait for all script results and returns EXIT_SUCCESS if all succeeded and failure otherwise